test/indentation-test-RRR.R

```diff
@@ -0,0 +1,349 @@
+
+
+### Function declarations
+
+## 1
+fun <- function(argument1,
+                argument2) {
+    body
+}
+
+## 2
+function(
+    argument1,
+    argument2
+    )
+    {
+        body
+    }
+
+## 3
+function(argument_fun(sub_argument1,
+                      sub_argument2),
+         argument) {}
+
+## 4
+function(argument1, parameter = fun_call(
+                        sub_argument),
+         argument2) {}
+
+
+
+### Function calls
+
+## 5
+fun_call(argument1,
+         argument2)
+
+## 6
+fun_call(
+    argument1,
+    argument2
+    )
+
+## 7
+fun_call(parameter = (
+    stuff
+    ),
+         argument)
+
+## 8
+fun_call(parameter = fun_argument(
+             stuff
+    ),
+         argument)
+
+
+
+### Blocks
+
+## 9
+function()
+    {
+        body
+    }
+
+## 10
+fun_call(parameter1 = {
+             stuff1
+         },
+         {
+             stuff2
+         }, parameter2 = {
+             stuff3
+         }, {
+             stuff4
+         },
+         parameter3 =
+             stuff5 ~
+                 stuff6 +
+                     stuff7)
+
+## 11
+fun <- fun_call({
+    stuff1
+}, {
+    stuff2
+},
+                {
+                    stuff3
+                }
+                )
+
+## 12
+fun_call(function(x) {
+             body1
+         },
+         function(x) {
+             body2
+         })
+
+## 13
+object <-
+    fun_call({
+        body
+    })
+
+## 14
+object <-
+    fun_call(     {
+                 body
+             }
+             )
+
+## 15
+{
+    {
+        stuff
+    }
+}
+
+## 16
+{{
+     stuff
+ }
+}
+
+## 17
+({
+    stuff
+})
+
+## 18
+( {
+     stuff
+ }
+ )
+
+## 19
+object[
+       argument1,
+       argument2
+       ]
+
+## 20
+{
+    object[
+           fun_call(
+               body
+               ),
+           argument[
+                    (
+                        sub_argument
+                        )
+                    ]
+           ]
+}
+
+
+
+### Control flow
+
+## 21
+if (condition) {
+    stuff1
+} else {
+    stuff2
+}
+
+## 22
+if (condition) {
+    stuff1
+}
+else {
+    stuff2
+}
+
+## 23
+if (condition)
+    {
+        stuff1
+    } else
+        {
+            stuff2
+        }
+
+## 24
+if (condition)
+    {
+        stuff1
+    }
+else
+    {
+        stuff2
+    }
+
+## 25
+for (sequence)
+    {
+        stuff
+    }
+
+## 26
+for (sequence) {
+    stuff
+}
+
+## 27
+if (condition)
+    stuff
+
+## 28
+for (sequence)
+    stuff
+
+
+
+### Continuation lines
+
+## 29
```

Comment:
I think it would wiser to start new numbering in each section. We will surely be adding stuff in the future and it would be nice to keep clean numbering without the need to modify everything else.